kotlin-css/src/commonMain/kotlin/kotlinx/css/CssBuilder.kt

```diff
@@ -190,8 +178,32 @@ fun String.toCustomProperty(): String {
     return "var(--$this)"
 }
 
+class CssBuilderImpl(
+    override val indent: String = "",
+    override val allowClasses: Boolean = true,
+    override val parent: RuleContainer? = null,
+) : CssBuilder {
+    override val classes = mutableListOf<String>()
+    override fun (RuleSet).unaryPlus() = this()
+
+    override val declarations = CSSDeclarations()
+
+    override var styleName = mutableListOf<String>()
+
+    override fun toString() = buildString {
+        declarations.forEach {
+            append("${it.key.hyphenize()}: ${it.value};\n")
+        }
+
+        buildRules(indent)
+    }
+
+    override val rules = mutableListOf<Rule>()
+    override val multiRules = mutableListOf<Rule>()
+}
+
 @Deprecated(
     message = "Legacy type alias",
     replaceWith = ReplaceWith("CssBuilder", "kotlinx.css.CssBuilder"),
 )
-typealias CSSBuilder = CssBuilder
+typealias CSSBuilder = CssBuilderImpl
```

Comment:
Why `CssBuilderImpl` and not `CssBuilder`?

Reply:
Thought it might be used somewhere as a constructor